src/JuMP.jl

```diff
@@ -735,7 +717,26 @@ include("macros.jl")
 include("optimizer_interface.jl")
 include("nlp.jl")
 include("print.jl")
+##########################################################################
```

Comment:
Nit: It really bothers me that there are 6 `#`s missing from this line.

Reply:
I removed all of these.